spec/graphql/mutations/post/create_post_spec.rb

```diff
@@ -51,6 +51,7 @@ def execute_query
 
     describe 'user is admin' do
       before do
+        allow(RestClient).to receive(:post)
```

Comment:
This is another smell if what I said above. Since the mutation is now coupled to Slack notifications, you have to stub things out in completely unrelated tests.